### PR TITLE
Don't require that JSX transformer be named JSXTransformer.js

### PR DESCRIPTION
**Pull request by chuckdumont**

### CODE REVIEW

**Comment by chuckdumont** [comment]
@cgorman

File: jaggr-core/src/main/java/com/ibm/jaggr/core/impl/resource/JsxResourceConverter.java [hunk 1]
```diff
@@ -527,7 +532,7 @@ public Scriptable call() throws Exception {
 							threadScope.setParentScope(null);
 							// require in the transformer and extract the transform function
 							Require require = builder.createRequire(ctx, threadScope);
-							Scriptable jsxTransformInstance = require.requireMain(ctx, JSXTRANSFORMER_NAME);
+							Scriptable jsxTransformInstance = require.requireMain(ctx, xformerModuleName);
```
Comment (ddumontatibm) [inline jaggr-core/src/main/java/com/ibm/jaggr/core/impl/resource/JsxResourceConverter.java]: Can we get rid of the constant as well or is it used elsewhere?

Comment (chuckdumont) [inline jaggr-core/src/main/java/com/ibm/jaggr/core/impl/resource/JsxResourceConverter.java]: It is also used as the name of the init-param that specifies the transformer URI

Comment (ddumontatibm) [inline jaggr-core/src/main/java/com/ibm/jaggr/core/impl/resource/JsxResourceConverter.java]: then this LGTM

**Pull request merged**